<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gamewindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/assets/profile.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/assets/profile.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AI Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Human Player</string>
    </property>
   </widget>
   <widget class="QComboBox" name="difficultyComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>easy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hard</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="firstplayComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>First Play</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Second Play</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="replayPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>RePlay</string>
    </property>
   </widget>
   <widget class="QWidget" name="boardWidget" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>552</width>
      <height>552</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="information">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
